ci/stf_tests.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: "Check that collectd is running and generating any metrics"
+  hosts: localhost
+  ignore_errors: true
+  become: true
+  environment:
+    PATH: "~/.crc/bin:~/.crc/bin/oc:~/bin:{{ ansible_env.PATH }}"
+  tasks:
+    - name: "Log into the cluster"
+      ansible.builtin.import_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+        
+    - name: "Update container_bin var"
+      tags:
+        - always
+      block:
+        - name: "Check for podman executable"
+          ansible.builtin.stat:
+            path: "/usr/bin/podman"
+          register: st_pod
+        - name: "Set container_bin=docker when podman isn't there"
+          ansible.builtin.set_fact:
+            container_bin: docker
+          when: not st_pod.stat.exists
+
+    - name: "Run collectd tests"
```

Comment:
We don't deploy collectd for 18, so this can be removed.

Unless you want some conditional execution depending on the version of openstack being used.
(I would recommend keeping conditional execution in mind, when making design decisions but not actually adding in anything conditional until the OSP17 job gets added)

Reply:
I'm to getting the list of required test for osp18 from leonid and I will remove all the unnecessary tasks untill I created the osp17 job